File: app/src/main/java/ir/saltech/answersheet/view/container/MaterialTextToast.java
package ir.saltech.answersheet.view.container;

import android.content.Context;

import androidx.annotation.NonNull;

public class MaterialTextToast extends MaterialToast {
    private long duration;

    public MaterialTextToast(@NonNull Context context) {
        super(context);
        super.init(getContentView());
    }

    public long getDuration() {
        return duration;
    }

    public void setDuration(long duration) {
        this.duration = duration;
    }

    public void showToast() {
        if (duration != 0) {
            super.show(duration);
        }
    }
}

File: app/src/main/java/ir/saltech/answersheet/view/container/MaterialTextToast.kt
package ir.saltech.answersheet.view.container

import android.content.Context

class MaterialTextToast(context: Context) : MaterialToast(context) {
    var duration: Long = 0

    init {
        super.init(contentView)
    }

    fun showToast() {
        if (duration != 0L) {
            super.show(duration)
        }
    }
}
